segments-guide.md

```diff
@@ -0,0 +1,329 @@
+
+## Authorization and Authentication
+
+To obtain authorization and authentication, see the [Getting Started Guide for 2.0 APIs](https://www.adobe.io/apis/experiencecloud/analytics/docs.html#!AdobeDocs/analytics-2.0-apis/master/create-oauth-client.md).
+
+## Best Practices
+
+Please follow these guidelines when using the Segments APIs:
+
+*  Make multiple, smaller requests instead of a large, single request.
+*  Request data once and cache it.
+*  Use caution when updating a segment so that you do not alter the original segment used by others.
+*  Avoid creating duplicate segments with the same definition. Creating many segments will affect performance for your company in some situations.
+*  If possible use the Analytics UI for creating segments from scratch. The UI offers significant advantages for managing and optimizing the complexity of this task. If you need to create segments programmatically, it is usually easier to create a template segment in the UI and then have your application change only small portions of the segment definition.
+
+## /segments Endpoint Description
+
+The `/segments` endpoint description is shown in our [Swagger UI](https://adobedocs.github.io/analytics-2.0-apis/). Use the Swagger UI to see endpoint summaries, available methods, parameters, example values, models, and status codes, and to try out the API.
+
+## Templates
+
+Adobe provides many predefined segments, or templates, for clients in specific situations. Although original templates cannot be modified, copies of them can be made and then the copies can be modified. If you want to work with segments templates programmatically, you can identify them by the additional attribute `template: true` as shown at the bottom of the following example:
+
+```
+
+{
+      "id": "Has_An_Action",
+      "name": "Has an Action",
+      "description": null,
+      "definition": {
+        "container": {
+          "func": "container",
+          "pred": {
+            "val": {
+              "func": "attr",
+              "name": "variables/mobileaction"
+            },
+            "func": "exists",
+            "description": "Action Name"
+          },
+          "context": "hits"
+        },
+        "func": "segment",
+        "version": [
+          1,
+          0,
+          0
+        ]
+      },
+      "template": true
+},
+
+```
+For more information on segments templates, see [Adobe Help](https://marketing.adobe.com/resources/help/en_US/analytics/segment/seg_build.html).
+
+### Expansions
+
+Segments endpoints support the URL query parameter `expansion`. This parameter allows specifying additional data fields to be populated in response objects. Available expansions are shown in the [Swagger UI](https://adobedocs.github.io/analytics-2.0-apis/) description. The following table describes expansion fields:
+
+|           Field       | 	Description         |
+|:---------------------:|:-------------------------:|
+| reportSuiteName | The name of the report suite
```

Comment:
It probably doesn't affect rendering, but we are missing closing pipes on the first and last lines here.

Reply:
I've now inserted those but it does not seem to be prompting me for a pull request for some reason.